app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_dict = {}
+
+    def wrapper(*args) -> int:
```

Comment:
- Good to add an annotation from the functools module
- Also, it should work with **kwargs. Create a unique key using args and kwargs
```suggestion
     @wraps(func)
    def wrapper(*args, **kwargs) -> int:
```

Reply:
@NazarSheremet could you please explain why you've rejected my solution? In course there are no mentions about @wraps(func)